app/src/pages/SingleContent.vue

```diff
@@ -117,45 +148,74 @@ const text = computed(() => {
+        <article class="mb-12 max-w-3xl lg:w-3/4">
+            <VideoPlayer v-if="content.video" :content="content" />
+            <img
+                v-else
+                :src="content.parentImage"
+                class="w-full rounded-lg object-cover shadow-md"
+            />
+
+            <h1 class="text-bold mt-4 text-center text-2xl text-zinc-800 dark:text-zinc-50">
+                {{ content.title }}
+            </h1>
+
+            <div
+                class="mt-1 text-center text-sm text-zinc-500 dark:text-zinc-300"
+                v-if="content.publishDate && content.parentPublishDateVisible"
+            >
+                {{ db.toDateTime(content.publishDate!).toLocaleString(DateTime.DATETIME_MED) }}
+            </div>
 
-        <div
-            v-if="content.text"
-            v-html="text"
-            class="prose prose-zinc mt-6 max-w-3xl text-justify dark:prose-invert"
-        ></div>
+            <div class="mt-12 text-justify text-gray-800 dark:text-zinc-100" v-if="content.summary">
+                {{ content.summary }}
+            </div>
 
-        <div
-            class="mt-6 border-t border-zinc-200 pt-6 dark:border-zinc-500"
-            v-if="content.parentTags.length > 0"
-        >
-            <h3 class="mb-2 text-sm text-zinc-600 dark:text-zinc-200">Tags</h3>
-            <div class="flex gap-3">
+            <div
+                v-if="content.text"
+                v-html="text"
+                class="prose prose-zinc mt-6 max-w-full text-justify dark:prose-invert"
+            ></div>
+        </article>
+
+        <div v-if="hasContent" class="h-full max-w-3xl py-2 lg:mt-0 lg:w-1/4">
+            <div
+                class="mb-5 flex flex-wrap border-b border-gray-200 text-center text-sm font-medium text-gray-500 dark:border-gray-700 dark:text-gray-400"
+            >
                 <span
-                    v-for="tag in tagsContent"
+                    v-for="tag in tagsContent.filter(
+                        (t: ContentDto) => t.parentTagType == TagType.Category,
+                    )"
                     :key="tag._id"
-                    class="inline-block rounded bg-yellow-300 px-1.5 py-1 text-sm text-yellow-950 shadow"
+                    @click="selectTag(tag.parentId)"
+                    class="me-2 flex cursor-pointer items-center justify-center rounded-t-sm px-1.5 py-1 text-sm hover:bg-yellow-100"
+                    :class="{ ' bg-yellow-400 text-black shadow': selectedTag == tag.parentId }"
                 >
                     {{ tag.title }}
                 </span>
             </div>
+
+            <div>
+                <VerticalTagViewer
+                    v-for="tag in tagCategory.filter(
+                        (t) => t.tagType == TagType.Category && t._id == selectedTag,
+                    )"
+                    :key="tag._id"
+                    :tag="tag"
+                    :currentPostId="content._id"
```

Comment:
This is called `currentPostId` but you're passing the `content._id`, so this should probably be named `currentContentId`?